kotlinx-coroutines-core/common/src/Job.kt

```diff
@@ -117,6 +117,22 @@ public interface Job : CoroutineContext.Element {
 
     // ------------ state query ------------
 
+    /**
+     * Returns a parent of the current job if the parent-child relationship
+     * is established or `null` if the job has no parent or was successfully completed.
+     *
+     * Accesses to this property are not idempotent, the property becomes `null` as soon
+     * as the job is transitioned to its final state, whether it is cancelled or completed,
+     * and all job children are completed.
```

Comment:
```suggestion
     * Accesses to this property are not idempotent: the property becomes `null` as soon
     * as the job is transitioned to its final state—whether it is cancelled or completed—
     * and all job children are completed.
```
Do we allow Unicode in comments?

Reply:
I would prefer to avoid it, we never know whether Dokka and kotlinlang support it

Reply:
This argument cuts both ways: we may treat this as a good chance to find out. I don't care that much though.